teeio-validator/library/helperlib/utils.c

```diff
@@ -531,3 +531,32 @@ bool get_uint32_array_from_string(uint32_t* array, uint32_t* array_size, const c
 
   return ret;
 }
+
+// dump key/iv stored in IDE-KM KEY_PROG message.
+// Refer to PCIe Spec 6.1 Figure 6-57
+void dump_key_iv_in_key_prog(const uint32_t *key, int key_dw_size, const uint32_t *iv, int iv_dw_size)
+{
+  uint32_t *key_buf = (uint32_t *)malloc(key_dw_size * sizeof(uint32_t));
```

Comment:
You can just use local variable.

no need to use malloc. malloc may fail and I dont see how that is handled.

Currently, you can assume AES-GCM-256, the key size is fixed.

Reply:
Updated.